CONTRIBUTING.md

```diff
@@ -202,7 +219,7 @@ Follow these commit guidelines:
 
   You may use [the `commit-msg` hook](./hooks/) to automatically sign-off your commits.
 
-- Follow the same naming schema as for [pull request titles](#pull-request-titles): `[type] #<issue number>: Description`. This means:
+- Follow the same naming schema as for [pull request titles](#pull-request-titles): `<type>: <description>`. This means:
```

Comment:
Mention that commit messages must follow Conventional Commits instead of constraining them to `<type>: <description>`